src/services/TestDataService.ts

```diff
@@ -473,7 +473,10 @@ export class TestDataService {
             data: basicCustomerStruct,
         });
 
-        const { data: customer } = (await response.json()) as { data: Customer };
+        const customerData = (await response.json()) as { data: Customer };
+
+        // @ts-ignore
+        const customer: Customer = { ...customerData.data, password: basicCustomerStruct.password };
```

Comment:
Someone now how to avoid the message which pops up from `password: basicCustomerStruct.password`

```
TS2322: Type string | undefined is not assignable to type string
Type undefined is not assignable to type string
```

Reply:
You have to ensure that the password exists... Because it can be `null`/`undefined` for guest customers. A simple check before this line should do the trick.

```ts
if (typeof basicCustomerDataStruc.password !== 'string')  {
    // fail the test
}
```

Reply:
Or even easier... You cast it to a string:

```suggestion
        const customer: Customer = { ...customerData.data, password: String(basicCustomerStruct.password) };
```